packages/expo-network/src/Network.ts

```diff
@@ -0,0 +1,36 @@
+import { Platform, UnavailabilityError } from '@unimodules/core';
+
+import ExpoNetwork from './ExpoNetwork';
+import { NetworkState } from './Network.types';
+
+export async function getNetworkStateAsync(): Promise<NetworkState> {
+  if (!ExpoNetwork.getNetworkStateAsync) {
+    throw new UnavailabilityError('expo-network', 'getNetworkStateAsync');
+  }
+  return await ExpoNetwork.getNetworkStateAsync();
+}
+
+export async function getIpAddressAsync(): Promise<string> {
+  if (!ExpoNetwork.getIpAddressAsync) {
+    throw new UnavailabilityError('expo-network', 'getIpAddressAsync');
+  }
+  return await ExpoNetwork.getIpAddressAsync();
+}
+
+export async function getMacAddressAsync(interfaceName?: string): Promise<string> {
+  if (!ExpoNetwork.getMacAddressAsync) {
+    throw new UnavailabilityError('expo-network', 'getMacAddressAsync');
+  }
+  if (Platform.OS === 'ios') {
```

Comment:
is it weirder for android to require an interfaceName, or weirder for iOS to not require one? (put another way, if we added another platform like windows, would you expect it to require an interfaceName or not?) whichever is weirder should be the one that is in the `if` statement. my hunch would be that this should be the other way around.